automation/tests/test_refactor.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from pathlib import Path
+print(Path)
+sys.path.append(str(Path(__file__).parent.parent.absolute()))
+
+import pytest
+import pytest_html
+
+from pages.landingPage import landingPage
+from pages.playPage import playPage
+
+from settings import settings
+
+# Test Parameters
+GoogleURL = "https://www.google.com/"
+
+@pytest.mark.usefixtures("driver_init")
+class BasicTest:
+    pass
+
+class Test_1(BasicTest):
```

Comment:
Probably worth thinking about how you can break up the tests into subsets of functionality.

Not sure if there is a standard convention in automation but you might want to describe the specific functionality being tested rather than gameplay/persistence. i.e. should_x_when_y. Don't need to resolve as part of this PR, we can readdress naming when you have a few more tests.